### PR TITLE
Obsolete: New Select part 3

### PR DESCRIPTION
**Pull request by NikoHelle**
See [the new version](https://github.com/City-of-Helsinki/helsinki-design-system/pull/1358), this kept for comments.

## Description

**Note: [Target branch is the part 2](https://github.com/City-of-Helsinki/helsinki-design-system/pull/1313) of the new Select!!**

Third part of the new Select.

Added:

- multiselect
- tags
- virtualization of long lists

## Related Issue

Closes [HDS-2069](https://helsinkisolutionoffice.atlassian.net/browse/HDS-2069)

## How Has This Been Tested?
Added unit tests

## Demos:

Links to demos are in the comments

## Add to changelog

Added in last PR

[HDS-2069]: https://helsinkisolutionoffice.atlassian.net/browse/HDS-2069?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/react/src/components/select/Select.module.scss [hunk 1]
```diff
@@ -222,18 +223,50 @@
...
-          cursor: pointer;
+        .singleSelectListItemLabel {
+          align-items: center;
+          display: flex;
           flex-grow: 1;
-          padding-left: var(--spacing-2-xs);
+          justify-content: space-between;
+
+          > span {
+            height: 24px;
```
Comment (laurakarhu) [inline packages/react/src/components/select/Select.module.scss:244]: should these be relative to font size?

File: packages/react/src/components/select/Select.module.scss [hunk 2]
```diff
@@ -291,6 +326,65 @@
...
+.tagListContainer {
+  display: flex;
+  flex-direction: column;
+
+  .tagList {
+    display: flex;
+    flex-flow: row wrap;
+    max-height: 78px;
+    overflow: hidden;
+    padding: 4px 0;
```
Comment (laurakarhu) [inline packages/react/src/components/select/Select.module.scss:338]: --spacing-3-xs ?

File: packages/react/src/components/select/Select.module.scss [hunk 3]
```diff
@@ -291,6 +326,65 @@
...
 }
 
+.tagListContainer {
+  display: flex;
+  flex-direction: column;
+
+  .tagList {
+    display: flex;
+    flex-flow: row wrap;
+    max-height: 78px;
```
Comment (laurakarhu) [inline packages/react/src/components/select/Select.module.scss:336]: Why 78px?

**Review by laurakarhu — COMMENTED** [review]
Couple of remarks related to sizes definitions. Nothing blocking but good to check.

File: packages/react/src/components/select/Select.module.scss (hunk 1 above)
Comment (NikoHelle) [reply packages/react/src/components/select/Select.module.scss:244]: No, this for an icon

File: packages/react/src/components/select/Select.module.scss (hunk 3 above)
Comment (NikoHelle) [reply packages/react/src/components/select/Select.module.scss:336]: It's the space needed to show two rows of tags. There is no constant for this, so have to hard code it.

File: packages/react/src/components/select/Select.module.scss (hunk 2 above)
Comment (NikoHelle) [reply packages/react/src/components/select/Select.module.scss:338]: Good point! Thx! Fixed.

**Pull request closed without merging**